include/APIClient/GPURequestor.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "restclient-cpp/restclient.h"
+#include "restclient-cpp/connection.h"
+
+#if defined(LUG_SYSTEM_WINDOWS)
+#pragma warning(push)
+#pragma warning(disable : 4003)
+#endif
+#include <json/json.hpp>
+#if defined(LUG_SYSTEM_WINDOWS)
+#pragma warning(pop)
+#endif
+
+#include "Router.hpp"
+
+using JSONResponse = std::tuple<int, nlohmann::json>;
+using ResponseCode = int;
+
+namespace APIClient {
+class GPURequestor {
```

Comment:
I'm not sure for the name of this class, it's redundant to specify `Requestor` because we are already in the namespace `APIClient`, so we know that it's something to do requests on the API.

Maybe `GPU` is just a better name? @Sumenia What do you think of that?